### PR TITLE
added documentation about default pull request description

### PR DESCRIPTION
**Pull request by montashevych**
# Checklist:

- [x] Green rspec run
- [x] Green rubocop
- [x] PR review from teammate

## What?
Default pull request template
## Why?
For more readability of the description
## How?
Added file PULL_REQUEST_TEMPLATE.md in folder .github
## Testing?
Only in master
## Screenshots (optional)
Don't need
## Anything Else?
-

### CODE REVIEW

**Review by KostyaKobel — APPROVED** [review]
All fine done

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## What?
+Explain the changes you’ve made. It doesn’t need to be fancy and you don’t have to get to technical, yet. Just explicit prose on your net change will typically suffice.
```
Comment (andrjuha01) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Add check marks for green rspec run, green rubocop, PR review from teammate

Comment (montashevych) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: done. but we will be able to see rspec and rubocop tests so because we have a github actions already.

**Review by andrjuha01 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (andrjuha01) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Almost forgot, checkmark: added specs for all new ruby code

Comment (montashevych) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: It's done)

**Pull request merged**